dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java

```diff
@@ -90,23 +88,23 @@ public static FieldSql getInstance() {
 	
 	
 	
-	public String inodeCount = "select count(inode) as inode_count from field where inode = ?";
+	public final String inodeCount = "select count(inode) as inode_count from field where inode = ?";
 	
 	
 	
-	public String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
+	public final String selectFieldOfDbType = "select field_contentlet from field where structure_inode = ? and field_contentlet like ? order by field_contentlet";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)